assets/src/scripts/components/hydrograph/graph-controls.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Hydrograph controls module.
+ */
+import { Actions } from '../../store';
+import { dispatch, link } from '../../lib/redux';
+import { audibleUI } from './audible';
+import { getCurrentVariableMedianStatistics } from '../../selectors/median-statistics-selector';
+import { isVisibleSelector, currentVariableTimeSeriesSelector } from './time-series';
+
+/*
+ * Create the show audible toggle, last year toggle, and median toggle for the time series graph.
+ * @param {Object} elem - D3 selection
+ */
+export const graphControls = function(elem) {
```

Comment:
Could we call this drawGraphControls? I think that is the pattern we have established and I like using verbs in function names.

Reply:
Yeah that's a great idea!